.github/workflows/filter-matrix.yaml

```
@@ -0,0 +1,47 @@
+name: "Filter Matrix Jobs"
+defaults:
+  run:
+    shell: bash
+on:
+  workflow_call:
+    inputs:
+      artifact:
+        required: true
+        type: string
+    outputs:
+      matrix:
+        description: "A JSON list containing the filtered jobs that can be run furhter"
```

an important contribution
```suggestion
        description: "A JSON list containing the filtered jobs that can be run further"
```